addon/ng2/blueprints/ng2/files/__path__/tsconfig.json

```diff
@@ -9,6 +9,12 @@
     "moduleResolution": "node",
     "outDir": "<%= relativeRootPath %>/dist/out-tsc",
     "sourceMap": true,
-    "target": "es5"
+    "target": "es5",
+    "typeRoots": [
+      "../node_modules/@types"
+    ],
+    "types": [
```

Comment:
So we'll have to have a discussion about this, and I need to talk to a bunch of other people. I'm against having jasmine typings for the _whole_ application. It should be for spec files only. But for that we would need to have 2 tsconfig files (or having different config). This was part of the discussions with the TypeScript team, and a solution does not exist yet. There are things that can be done though.